provisioning/standalone/docs/example.md

```diff
@@ -0,0 +1,94 @@
+# Example
+
+In this example, we are going to learn how to deploy a simple echo server using **golang**, which will answer to a **GET HTTP request** with a predefined message.
+
+### Create the server
+
+Create a **main.go** file which will contain the source code of the server and **copy/paste** the following code inside **main.go**
+
+```go
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello!\n")
+}
+
+func main() {
+	port := os.Args[1]
+	basepath := os.Args[2]
+	http.HandleFunc(fmt.Sprintf("%s/", basepath), hello)
+	err := http.ListenAndServe(fmt.Sprintf("0.0.0.0:%s", port), nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+}
+```
+
+Let's test it by running it with `go run 8888 /hello`.
+
+In the meanwhile open another terminal and launch the command `curl localhost:8888/hello`, and you will get `Hello!` as response.
+
+### Build the container
+
+Let's create a **Dockerfile** ` and **copy/paste** the following code inside of it.
+
+```dockerfile
+FROM golang:1.18
+ENV CROWNLABS_LISTEN_PORT=8888
+ENV CROWNLABS_BASE_PATH=/basepath
+COPY main.go main.go
+RUN go build main.go
+RUN useradd -ms /bin/bash myuser
+USER myuser
+CMD exec ./main ${CROWNLABS_LISTEN_PORT} ${CROWNLABS_BASE_PATH}
+```
+
+Build the container with `docker build -t ${YOUR_DOCKERHUB_USERNAME}/echotest` and push it on **DockerHub** with `docker push ${YOUR_DOCKERHUB_USERNAME}/echotest`.
+
+### Create the template
+
+Create a file called **template.yaml** and **copy/paste** the following code:
+
+```yaml
+apiVersion: crownlabs.polito.it/v1alpha2
+kind: Template
+metadata:
+    name: echotest
+    namespace: workspace-sid # a workspace where you have the permission to create templates
+spec:
+    deleteAfter: never
+    description: An http server used for testing
+    environmentList:
+        - environmentType: Standalone
+          guiEnabled: true
+          image: YOUR_DOCKERHUB_USERNAME/echotest
+          mode: Standard
+          name: default
+          persistent: false
+          resources:
+              cpu: 1
+              memory: 1000M
+              reservedCPUPercentage: 50
+          rewriteURL: false
+    prettyName: Echo HTTP Server
+    workspace.crownlabs.polito.it/WorkspaceRef:
+        name: sid # choosen workspace name
+```
+
+And **apply** it with `kubectl apply -f template.yaml`
+
+### Create an instance and try it
+
+Open **CrownLabs dashboard** and create a new instance.
+
+<img align="left" src="./img.png">
+<br clear="left"/>
+<br clear="left"/>
```

Comment:
Is it there any reason for not using the standard markdown syntax?

Reply:
I did it to justify the image on the left, I cannot find another solution.

Reply:
I would personally favor simplicity using the standard approach.

Reply:
Ok 👍